seed/csharp-sdk/seed.yml

```diff
@@ -2,20 +2,29 @@ displayName: C# SDK
 irVersion: v53
 image: fernapi/fern-csharp-sdk
 changelogLocation: ../../generators/csharp/sdk/versions.yml
+
+artifactMetadata:
```

Comment:
could we rename this to `buildScripts` ? think it communicates the meaning a bit more

Reply:
most definitely!